module/dice.js

```diff
@@ -309,22 +309,28 @@ export function getAttributeAndSkill(skillName, data, attributeName = null) {
  * @param {number}  [modifier=0]   The task modifier
  * @param {boolean} [locate=false] Whether to roll a Location die
  * @see {YearZeroRoll}
- * @returns {import('../lib/yzur.js').DiceQuantities}
+ * @returns {Array.<import('../lib/yzur.js').TermBlok>}
  */
 export function getDiceQuantities(attribute, skill = 0, rof = 0, locate = false) {
```

Comment:
Note: changes in `dice.js: getDiceQuantities()` were not mandatory because the function `YearZeroRoll.forge()` still accepts the old format parameter (with a deprecation warning).